packages/strapi-plugin-upload/admin/src/containers/HomePage/saga.js

```diff
@@ -81,8 +81,11 @@ function* uploadFiles(action) {
       strapi.notification.success({ id: 'upload.notification.dropFiles.success', values: { number: newFiles.length } });
     }
 
-  } catch(err) {
-    strapi.notification.error('notification.error');
+  } catch(error) {
+    let message = get(error, ['response', 'payload', 'message', '0', 'messages', '0']);
+    if (isObject(message)) message = {...message, id: `upload.${message.id}`};
```

Comment:
Can you make the following update so we don't have to specify the name of the plugin?

```
import pluginId from '../../pluginId';

if (isObject(message)) {
  message = { ...message, id: `${pluginId}.${message.id}` };
}
```

Reply:
@soupette done